packages/caliper-core/test/manager/monitors/monitor-docker.js

```diff
@@ -0,0 +1,211 @@
+            });
+        });
+
+        it('should add and label containers as local', async () => {
+            const containers = [
+                {
+                    id:"118",
+                    name:"peer0.org1.example.com",
+                    remote:null
+                }
+            ]
+            await dockerMonitor.findContainers().then(()=>{
+                dockerMonitor.containers.should.be.an('array').that.deep.equals(containers);
+            });
+        });
+
+        /*it('should add and label containers as remote', async () => {
+            await dockerMonitor.findContainers().then(()=>{
+                for(let key in this.options.containers){
+                    dockerMonitor.containers[i].name
+                }
+            });
+        });*/
+    });
+
+    describe('whan reading statistics', () => {
+
+        //memory test values
+        const mem_usage = dockerStats.memory_stats.usage - dockerStats.memory_stats.stats.cache;
+        const mem_percent = mem_usage / dockerStats.mem_limit;
+
+        beforeEach( () => {
+            dockerMonitor = new DockerMonitorRewire(monitorOptions);
+        });
+
+        it('should set isReading to false', async () => {
+            await dockerMonitor.findContainers().then(async () => {
+                await dockerMonitor.readContainerStats()
+            }).then(async () => {
+                dockerMonitor.isReading.should.be.false;
+            });
+        });
+
+        it('should read and add stats for each target container', async () => {
+
+            await dockerMonitor.findContainers().then(async () => {
+                await dockerMonitor.readContainerStats()
+            }).then(async () => {
+                dockerMonitor.stats["118"].mem_usage.should.be.an('array').that.deep.equals([mem_usage]);
+            });
+        });
+
+    });
+
+    describe('when getting statistics', () => {
+
+        beforeEach( () => {
+            dockerMonitor = new DockerMonitorRewire(monitorOptions);
+        });
+
+        const cpuCores = os.cpus().length;
+
+        //cpu percentage test metric calculation
+        let cpuDelta = dockerStats.cpu_stats.cpu_usage.total_usage - dockerStats.precpu_stats.cpu_usage.total_usage;
+        let sysDelta = dockerStats.cpu_stats.system_cpu_usage - dockerStats.precpu_stats.system_cpu_usage;
+        let cpu_percent;
+        if (cpuDelta > 0 && sysDelta > 0) {
+            if (dockerStats.cpu_stats.cpu_usage.hasOwnProperty('percpu_usage') && dockerStats.cpu_stats.cpu_usage.percpu_usage !== null) {
+                cpu_percent = (cpuDelta / sysDelta * cpuCores * 100.0);
+            } else {
+                cpu_percent = (cpuDelta / sysDelta * 100.0);
+            }
+        } else {
+            cpu_percent = 0;
+        }
+
+        it('should normalize cpuUsage if cpuUsageNormalization property is present', async () => {
```

Comment:
There should be 2 more tests
1. should not normalize if property set to false
2. should not normalize if property not provided

test should say `should normalize cpu usage if cpuUsageNormalization property is set to true`